cube/cube.cpp

```diff
@@ -703,69 +610,58 @@ void Demo::cleanup() {
     dfb->Release(dfb);
 #endif
 
-    inst.destroy(nullptr);
+    inst.destroy();
 }
 
 void Demo::create_device() {
-    float const priorities[1] = {0.0};
-
-    vk::DeviceQueueCreateInfo queues[2];
-    queues[0].setQueueFamilyIndex(graphics_queue_family_index);
-    queues[0].setQueueCount(1);
-    queues[0].setPQueuePriorities(priorities);
-
-    auto deviceInfo = vk::DeviceCreateInfo()
-                          .setQueueCreateInfoCount(1)
-                          .setPQueueCreateInfos(queues)
-                          .setEnabledLayerCount(0)
-                          .setPpEnabledLayerNames(nullptr)
-                          .setEnabledExtensionCount(enabled_extension_count)
-                          .setPpEnabledExtensionNames((const char *const *)extension_names)
-                          .setPEnabledFeatures(nullptr);
+    float priorities = {0.0};
```

Comment:
The braces around `0.0` seem unnecessary (and I think they'll break on older Linux compilers).

Reply:
Yep, stale code. fixed.